examples/terminal/terminal

```diff
@@ -50,26 +50,37 @@ MAIN_DIALOG='
 				<action signal="child-exited">echo "vte0=$vte0 - New PID"</action>
 				<action signal="show">echo "vte0=$vte0 - show signal"</action>
 			</terminal>
-			<terminal argv0="/bin/sh"
+			<terminal argv0="'${SHELL:-/bin/sh}'"
 				envv0="WELCOME=Welcome to the terminal (VteTerminal) widget :)"
 				pointer-autohide="true"
 				current-directory-uri="'$HOME'"
-				cursor-background-color="#ff0"
-				highlight-background-color="#660"
-				text-background-color="#332"
-				text-foreground-color="#fff"
+
+				cursor-background-color="aqua"
+				highlight-background-color="#808000"
+				highlight-foreground-color="red"
+				text-background-color="#44d"
+				text-foreground-color="#ddd"
+
 				hscrollbar-policy="2"
 				vscrollbar-policy="2">
 				<variable>vte1</variable>
-				<input>echo echo \$WELCOME</input>
+
+				<input>echo "_info() { printf \"\033[48;2;%d;%d;%dm     \033[0m %s\n\" \"\$@\"; }"</input>
+				<input>echo " echo;echo;echo; echo \"\$WELCOME\"; echo;echo;echo"</input>
```

Comment:
You may want to prevent these first two lines from showing up in the terminal, by adding this line below them:

`<input>echo "printf \"\033c\""</input>`

Unless you want the user to see what's going on under the hood.

Reply:
Good idea, thank you!

Reply:
I didn't notice at first, but it also hides the "WELCOME" message, so maybe only the first line...

Reply:
Hmm, I tried but with `SHELL=dash` the welcome message disappears, and with `SHELL=bash` I get this
![gtk3-20220307](https://user-images.githubusercontent.com/1414102/157126682-c20e7f46-c535-4ebc-9851-5118b20fef47.png)

It appears that output format depends on the user shell.  At this point I don't want to spend more time on this demo script. I could revert commit a6ce45d, what do you think?

Reply:
I just noticed that I can only get the output like on the screenshot above, with any shell.
Can't make it to look like on your previous screenshots.
So yeah, maybe just revert it for now...

Reply:
This is the command that produced the first screenshot in https://github.com/puppylinux-woof-CE/gtkdialog/pull/137#discussion_r821135156
```sh
SHELL=dash GTKDIALOG=gtkdialog_gtk3 ./examples/terminal/terminal
```
The olive background in that screenshot is because I selected some text.

Reply:
Oh, I see. Still can't get the two-block appearance, though:

![xscreenshot-20220307T235323](https://user-images.githubusercontent.com/6226504/157131921-4dc54fd3-93e8-4062-bade-d5ccfa0a96ab.png)